apps/meteor/app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -130,7 +130,9 @@ export class CustomOAuth {
 		}
 
 		try {
-			response = HTTP.post(this.tokenPath, allOptions);
+			const queryparams = new URLSearchParams(allOptions.params);
+			const request = await fetch(`${this.tokenPath}?${queryparams.toString()}`, { method: 'POST', headers: allOptions.headers });
+			response = await request.json();
```

Comment:
Response from `HTTP.post` is different from the `json()` result right? I believe it's the data only and the response variable used below is trying to get data from `response.data`.